lib/rouge/lexers/lustre.rb

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*- #
+#
+# adapted from ocaml.rb, hence some ocaml-ism migth remains
+module Rouge
+  module Lexers
+    class Lustre < RegexLexer
+      title "Lustre"
+      desc 'The Lustre programming language (Verimag)'
+      tag 'lustre'
+      filenames '*.lus'
+      mimetypes 'text/x-lustre'
+
+      def self.keywords
+        @keywords ||= Set.new %w(
+          extern unsafe assert const current enum function
+          false let node operator returns
```

Comment:
Sorry, I missed this in the initial review but `false` and `true` should be removed from this list.

Reply:
ok.